app/js/csvAddShow.js

```diff
@@ -66,7 +65,22 @@ document.getElementById('chooseCSV').addEventListener('click', () => {
                 const newColumn = document.createElement('td');
                 newColumn.appendChild(associateAudioButton);
 
-                const timeFrameLengthInput = document.createElement('input');
+                const timeFrameLengthInput = document.createElement('select');
+                const option25  = document.createElement('option');
+                const option50  = document.createElement('option');
+                const option100 = document.createElement('option');
+
+                option25.text   = 25;
+                option25.value  = '25';
+                option50.value  = 50;
+                option50.text   = '50';
+                option100.value = 100;
+                option100.text  = '100';
```

Comment:
Is there a point to have multiple spaces to line up the assignment operators? Seems like a frivolous thing to do for only one part of the whole application